### PR TITLE
Bug 1490826 - Add form info to FxA form submits

### PR DESCRIPTION
**Pull request by vladikoff**
cc @ncloudioj 

Context: https://bugzilla.mozilla.org/show_bug.cgi?id=1490826#c9

### CODE REVIEW

**Comment by vladikoff** [comment]
>  Could you add those events to data_events.md? 

@ncloudioj updated!

**Comment by ncloudioj** [comment]
@vladikoff r+

Now let's do the data review before merging it.

File: content-src/components/StartupOverlay/StartupOverlay.jsx [hunk 1]
```diff
@@ -69,15 +71,23 @@ export class _StartupOverlay extends React.PureComponent {
   }
 
   onSubmit() {
-    this.props.dispatch(ac.UserEvent({event: "SUBMIT_EMAIL"}));
+    this.props.dispatch(ac.UserEvent(Object.assign({event: "SUBMIT_EMAIL"}, this._getFormInfo())));
```
Comment (k88hudson) [inline content-src/components/StartupOverlay/StartupOverlay.jsx]: `{event: "SUBMIT_EMAIL", ...this._getFormInfo()}` is fine here

Comment (ncloudioj) [reply content-src/components/StartupOverlay/StartupOverlay.jsx]: Out of curiosity, what is the main benefit of this form compared to Object.assign?

Comment (k88hudson) [reply content-src/components/StartupOverlay/StartupOverlay.jsx]: no benefit, personally I just think it's easier to read :)

Comment (vladikoff) [reply content-src/components/StartupOverlay/StartupOverlay.jsx]: it's also more hip, but I tried to stay consistent with the repo

Comment (vladikoff) [reply content-src/components/StartupOverlay/StartupOverlay.jsx]: (just to clarify that doesn't mean your repo ain't hip)

**Comment by ncloudioj** [comment]
Data review r+ granted at https://bugzilla.mozilla.org/show_bug.cgi?id=1490826#c15

**Comment by vladikoff** [comment]
Amazing! Thank you for reviews and merge!

**Pull request merged**